.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         run: cargo test --no-fail-fast --all-targets --all-features --workspace
 
       - name: Async-std Test
-        run: cargo test --no-fail-fast --all-targets --no-default-features  --features "async-std" --features "storage-fs" --workspace
+        run: cargo test --no-fail-fast --all-targets --no-default-features  --features "async-std" --features "storage-all" --workspace
```

Comment:
I'm not 100% certain this is the right approach, but I figured rather than adding on another `--features "storage-gcs"`, simply using `storage-all` here covers it for future contributors too.

Granted, I'm quite sure whether this is what we want. Happy to alter/take suggestions :+1:

Reply:
I'm ok with this in CI.